frontend/src/components/static_pages/phoneNumber.jsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { connect } from "react-redux";
+import getStrapi from "../../actions/strapi.js";
+
+class PhoneNumber extends React.Component {
+  componentDidMount() {
+    this.props.gPhone();
+  }
+
+  render() {
+    return <>{this.props.phone}</>;
+  }
+}
+
+const mapStateToProps = state => {
+  return {
+    phone: state.strapi.number
```

Comment:
this needs to be `.phone` not `.number`

Reply:
it's "number" in Strapi, sorry for the confusion.

Reply:
ahhh it needs to be updated in the reducer. For [initialState](https://github.com/the-difference-engine/ymim/blob/strapi-posgres/frontend/src/reducers/strapi.js#L19) -> we have phone but for the action we set number.